src/types/domain.d.ts

```diff
@@ -234,3 +234,29 @@ export interface ReferenceData {
   themes?: EuTheme[];
   referencetypes?: ReferenceType[];
 }
+
+export interface Link {
+  href: string;
+}
+export interface Links {
+  next: Link;
+  self: Link;
+}
+
+export interface NewsItemAttributes {
+  title: string;
+  created: string;
+  changed: Date;
+  field_ingress: string;
+}
+
+export interface NewsItem {
+  type: string;
+  id: string;
+  links: Partial<Links>;
+  attributes: Partial<NewsItemAttributes>;
+}
+
+export interface NewsItemList {
+  data: NewsItem[];
```

Comment:
Unødvendig data attributt - vi burde mappe den over til News[] direkte

Reply:
Vi får strukturen fra ekstern CMS, det vil etterhvert også bli bruk for 'links' ved siden av 'data' så vi bør vel ikke mappe den direkte.

Reply:
Fjernet